server/settings.py

```diff
@@ -54,6 +54,7 @@ def env(variable, fallback_value=None):
     'ntb.scanpix',
     'planning',
     'ntb.io',
+    'ntb.io.feeding_services.newsworthy',
```

Comment:
Why do we have a separate app for it?

Reply:
Because I need to access config to set the URL at https://github.com/superdesk/superdesk-ntb/pull/309/files#diff-eb7617cf8ee7bfaf4e73b92c7f33c043R152 and this is not possible at parsing time. With an app, `init_app` is called where we can access config.